server/pulp/plugins/rsync/publish.py

```diff
@@ -258,6 +258,36 @@ def process_main(self):
             raise PulpCodedException(message=output)
 
 
+class UpdateLastPredistDateStep(PublishStep):
```

Comment:
So, it's currently only used in the RPM plugin, but should I go add this to docker as well? I haven't tested to see whether the docker rsync distributor is vulnerable to the same unit skipping issue.